src/main/java/mate/jdbc/dao/DriverDaoImpl.java

```diff
@@ -0,0 +1,118 @@
+package mate.jdbc.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.lib.exception.DataProcessingException;
+import mate.jdbc.model.Driver;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class DriverDaoImpl implements DriverDao {
+    @Override
+    public Driver create(Driver driver) {
+        String createRequest = "INSERT INTO drivers (name, license_number) "
+                + "VALUES (?, ?)";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement createDriverStatement
+                        = connection.prepareStatement(createRequest,
+                                Statement.RETURN_GENERATED_KEYS)) {
+            createDriverStatement.setString(1, driver.getName());
+            createDriverStatement.setString(2, driver.getLicenseNumber());
+            createDriverStatement.executeUpdate();
+            ResultSet resultSet = createDriverStatement.getGeneratedKeys();
+            if (resultSet.next()) {
+                driver.setId(resultSet.getObject(1, Long.class));
+            }
+            return driver;
+        } catch (SQLException throwable) {
+            throw new DataProcessingException("Couldn't create driver " + driver + " in DB ",
+                    throwable);
+        }
+    }
+
+    @Override
+    public Optional<Driver> get(Long id) {
+        String getRequest = "SELECT * FROM drivers"
+                + " WHERE id = (?) AND is_deleted = FALSE";
```

Comment:
Why do we need brackets here?
```suggestion
                + " WHERE id = ? AND is_deleted = FALSE";
```

Reply:
not need)